package.json

```diff
@@ -20,10 +20,10 @@
     "ember-auto-focus": "1.0.9",
     "ember-autoresize": "0.6.0",
     "ember-can": "^0.8.4",
-    "ember-cli": "2.13.1",
+    "ember-cli": "~2.14.0-beta.2",
     "ember-cli-app-version": "^3.0.0",
     "ember-cli-autoprefixer": "0.7.0",
-    "ember-cli-babel": "^5.2.4",
+    "ember-cli-babel": "^6.4.1",
```

Comment:
`ember init` sets the version to `6.0.0`, but really, the problem was the transition from `5.x` to `6.x`, so no point not to upgrade further.